websocket.go

```diff
@@ -68,4 +69,11 @@ func wsHandler(errCh chan error, hub *Hub, w http.ResponseWriter, r *http.Reques
 		}
 	}(errCh)
 
+	go func() {
+		for {
+			msg := <-jsonReqChan
+			writer(conn, msg)
```

Comment:
we should not use `writer` this way; `writer` should work like `reader` ; smth like:
```go
func writer(conn conn *websocket, messages chan []byte) {
  for {
    msg <- messages
    err := conn.WriteMessage(1, msg)
    ...
  }
}
```
then you could just write messages to corresponding chan and be sure that writer will do its job